src/spring-cloud/azext_spring_cloud/custom.py

```diff
@@ -1344,18 +1349,124 @@ def get_logs_loop():
                        resource_group, service, app, name, properties=properties, sku=sku)
 
 
-def _get_app_log(url, user_name, password, exceptions):
+# pylint: disable=bare-except, too-many-statements
+def _get_app_log(url, user_name, password, format_json, exceptions):
+    logger_seg_regex = re.compile(r'([^\.])[^\.]+\.')
+
+    def build_log_shortener(length):
+        if length <= 0:
+            raise CLIError('Logger length in `logger{length}` should be positive')
```

Comment:
Please use the specific error type instead of `CLIError`，such as `InvalidArgumentValueError`

Reply:
got it, updated.